snapcraft/tests/test_commands_pull.py

```diff
@@ -50,14 +50,25 @@ def make_snapcraft_yaml(self, n=1):
         parts = []
         for i in range(n):
             part_dir = os.path.join(common.get_partsdir(), 'pull{}'.format(i))
-            state_file = os.path.join(part_dir, 'state')
+            state_dir = os.path.join(part_dir, 'state')
             parts.append({
                 'part_dir': part_dir,
-                'state_file': state_file,
+                'state_dir': state_dir,
             })
 
         return parts
 
+    def verify_state(self, part_name, state_dir, expected_step):
```

Comment:
This looks like copy paste code, why not put it as part of the general test class or something? @elopio can provide guidance